crates/config/src/revive.rs

```diff
@@ -41,7 +38,6 @@ impl ResolcConfig {
             .sources(&config.src)
             .tests(&config.test)
             .scripts(&config.script)
-            .artifacts(config.root.join(RESOLC_ARTIFACTS_DIR))
```

Comment:
should not we set  `.artifacts(&self.out)`?

Reply:
the link does not work on my side

Reply:
here's non borked one [click me](https://github.com/paritytech/foundry-revive-compiler/blob/master/crates/compilers/src/config.rs#L895)